pkg/report/linux.go

```diff
@@ -759,6 +765,11 @@ func (ctx *linux) getMaintainers(file string) (vcs.Recipients, error) {
 	if ctx.kernelSrc == "" {
 		return nil, nil
 	}
+
+	if _, err := os.Stat(filepath.Join(ctx.kernelSrc, maintainerScript)); errors.Is(err, os.ErrNotExist) {
```

Comment:
I am a bit afraid that this can silently break upstream linux reporting (if the script is renamed or something). If that happens bugs will be reported to /dev/null and we will need to fix up database.

It will be safer if you add VM type to the config struct, check it here and skip get_maintainers for android only.
I also suspect we may need more customization for android in future (modules, different path, etc).